posts/2023-01-20-types-announcement.md

```diff
@@ -0,0 +1,120 @@
+
+Another relevant working group is the [polonius working group](https://rust-lang.github.io/compiler-team/working-groups/polonius/), which primarily works on the design and implementation of the [Polonius](https://github.com/rust-lang/polonius) borrow-checking library. While the working group itself will remain, it is now also under the purview of the types team.
+
+Now, although the traits working group was essentially folded into the types team, the creation of a *team* has some benefits. First, like the [style team](https://blog.rust-lang.org/inside-rust/2022/09/29/announcing-the-rust-style-team.html) (and many other teams), the types team is not a *top level* team. It actually, currently uniquely, has *two* parent teams: the lang and compiler teams. Both teams have decided to delegate decision-making authority covering the type system.
+
+The language team has delegated the part of the *design* of type system. However, importantly, this design covers less of the "feel" of the features of type system and more of how it "works", with the expectation that the types team will advise and bring concerns about new language extensions where required. (This division is not strongly defined, but the expectation is generally to err on the side of more caution). The compiler team, on the other hand, has delegated the responsibility of defining and maintaining the implementation of the trait system.
+
+One particular responsibility that has traditionally been shared between the language and compiler teams is the assessment and fixing of soundness bugs in the language related to the type system. These often arise from implementation-defined language semantics and have in the past required synchronization and input from both lang and compiler teams. In the majority of cases, the types team now has the authority to assess and implement fixes without the direct input from either parent team. This applies, importantly, for fixes that are *technically* backwards-incompatible. While fixing safety holes is [not covered under Rust's backwards compatibility guarantees](https://blog.rust-lang.org/2014/10/30/Stability.html#what-are-the-stability-caveats), these decisions are not taken lightly and generally require team signoff and are assessd for potential ecoystem breakage with [crater](https://github.com/rust-lang/crater). However, this can now be done under one team rather than requiring the coordination of two separate teams, which makes closing these soundness holes easier (I will discuss this more later.)
```

Comment:
```suggestion
One particular responsibility that has traditionally been shared between the language and compiler teams is the assessment and fixing of soundness bugs in the language related to the type system. These often arise from implementation-defined language semantics and have in the past required synchronization and input from both lang and compiler teams. In the majority of cases, the types team now has the authority to assess and implement fixes without the direct input from either parent team. This applies, importantly, for fixes that are *technically* backwards-incompatible. While fixing safety holes is [not covered under Rust's backwards compatibility guarantees](https://blog.rust-lang.org/2014/10/30/Stability.html#what-are-the-stability-caveats), these decisions are not taken lightly and generally require team sign off and are assessed for potential ecoystem breakage with [crater](https://github.com/rust-lang/crater). However, this can now be done under one team rather than requiring the coordination of two separate teams, which makes closing these soundness holes easier (I will discuss this more later.)
```